README.md

```diff
@@ -1226,6 +1212,25 @@ ESLint rules (applies this guide as linting rules):
     };
   };
   ```
+  
+  > NOTE: You should group all your `const`s and then afterward group all your `let`s if declared in vicinity of one another. This is helpful when later on you might need to assign a variable depending on one of the previous assigned variables.
+
+  ```javascript
+  // bad
+  let i;
+  const items = getItems();
+  let dragonball;
+  const goSportsTeam = true;
+  let len;
+
+  // good
+  const goSportsTeam = true;
+  const items = getItems();
+
+  let dragonball;
+  let i;
```

Comment:
I know this is just illustrating organization of lets and consts, but because we don't use single letter or abbreviated variable names, perhaps we should not show them being used in our //good sections

Reply:
good call, ill update